lib/types.ts

```diff
@@ -64,3 +64,49 @@ export interface DepTree {
     [dep: string]: DepTree;
   };
 }
+
+export interface GoPackage {
```

Comment:
a link to the original code/documentation would be nice

Reply:
Will try to find, this is taken from the output of `go help list` command.